apps/android/app/src/main/java/chat/simplex/app/views/chat/item/FramedItemView.kt

```diff
@@ -131,15 +132,15 @@ fun FramedItemView(
       Column(Modifier.width(IntrinsicSize.Max)) {
         PriorityLayout(Modifier, CHAT_IMAGE_LAYOUT_ID) {
           ci.quotedItem?.let { ciQuoteView(it) }
-          if (ci.file == null && ci.formattedText == null && isShortEmoji(ci.content.text)) {
```

Comment:
is it always the same?

Reply:
I am not sure why this change is needed, and I am actually not sure it is the same.